scanners/dispatcher/dispatcher.py

```diff
@@ -1,179 +1,108 @@
-import os
+import ast
+import emoji
 import sys
 import logging
 import requests
-import jwt
-import threading
-from flask import Flask, request
+from starlette.applications import Starlette
```

Comment:
This is all I want for Christmas, but you know who says no :sob:

Reply:
😂